docs/x509/reference.rst

```diff
@@ -462,6 +462,24 @@ X.509 Certificate Object
        An :class:`~cryptography.exceptions.InvalidSignature` exception will be
        raised if the signature fails to verify.
 
+
+    .. attribute:: tbs_precertificate_bytes
+
+        .. versionadded:: 38.0
+
+        :type: bytes
+
```

Comment:
The docs should provide more context here, imagine a user who isn't familiar with CT reading this. It doesn't need to explain all of CT, but it does need to provide enough information for them to figure out if this is relevant to them.